src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,51 @@
-export const Tabs = () => {};
+import cn from 'classnames';
+import { Tab } from '../../react-app-env';
+
+type Props = {
+  tabs: Tab[];
+  selectedTabId: string;
+  onTabSelected: (tab: Tab) => void;
+};
+
+export const Tabs: React.FC<Props> = ({
+  tabs,
+  selectedTabId,
+  onTabSelected,
+}) => {
+  const currentTab = tabs.some(tab => selectedTabId === tab.id);
```

Comment:
I'd rather choose current tab using find method and use this variable further in the code
```suggestion
  const currentTab = tabs.find(tab => selectedTabId === tab.id) || tabs[0;
```